### PR TITLE
Here are some changes related with the specific functionality

### PR DESCRIPTION
**Pull request by tklepacki**
Some description

### CODE REVIEW

File: src/test/java/mohito/pl/LoginTest.java [hunk 1]
```diff
@@ -38,13 +38,11 @@ public void loginTest(String email, String password, String firstName) {
 		commonTestSteps.loadMainPage(URL);
 		driver.findElement(By.id("cookiebotDialogOkButton")).click();
 		commonTestSteps.loginProcess(email, password);
-
 		assertEquals(firstName, manager.getUserMenuPage().getAccountNameText(firstName));
 	}
```
Comment (Automatyzacja-IT) [inline src/test/java/mohito/pl/LoginTest.java:42]: This is bad

File: src/test/java/mohito/pl/LoginTest.java [hunk 2]
```diff
@@ -38,13 +38,11 @@ public void loginTest(String email, String password, String firstName) {
...
 
 	@AfterMethod
 	public void tearDown() {
 		driver.quit();
 	}
-
```
Comment (Automatyzacja-IT) [inline src/test/java/mohito/pl/LoginTest.java:49]: This is also bad

File: src/test/java/mohito/pl/LoginTest.java (hunk 1 above)
Comment (tklepacki) [reply src/test/java/mohito/pl/LoginTest.java:42]: No its not

File: src/test/java/mohito/pl/LoginTest.java (hunk 2 above)
Comment (tklepacki) [reply src/test/java/mohito/pl/LoginTest.java:49]: No it's not

**Pull request merged**